### PR TITLE
Make convenience boxed true/false easier to use

### PR DESCRIPTION
**Pull request by jwhonce**
Small improvement for working with the API bindings.

* changed PFalse to &false
* changed PTrue to &true

Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

File: pkg/bindings/bindings.go [hunk 1]
```diff
@@ -11,8 +11,10 @@ package bindings
 var (
 	// PTrue is a convenience variable that can be used in bindings where
 	// a pointer to a bool (optional parameter) is required.
-	PTrue bool = true
+	pTrue = true
+	PTrue = &pTrue
 	// PFalse is a convenience variable that can be used in bindings where
```
Comment (TomSweeneyRedHat) [inline pkg/bindings/bindings.go:16]: PFalse and PTrue in comments need to change to pFalse and pTrue

Comment (TomSweeneyRedHat) [reply pkg/bindings/bindings.go:16]: oops, never mind, I should look at line two for each of these.

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests

File: pkg/bindings/bindings.go (hunk 1 above)
Comment (jwhonce) [reply pkg/bindings/bindings.go:16]: @TomSweeneyRedHat The comments are more meaningful now. We don't want people using pTrue or pFalse.

**Pull request merged**